### DISCUSSION TITLE
Solving Inverse Problem with  non Linear PDE

### DISCUSSION THREAD

**Discussion by ppandit95** [lululxvi/deepxde#830] [Q&A]
Dear DeepXDE Developer Community,

I wish to use DeepXDE for solving inverse problem but the PDE involved is non linear in nature wherein my unknown parameter depends on the solution variable . So it will be of tremendous help if anybody can suggest a strategy to use neural networks such that I can train unknown parameter when I do not have the knowledge of underlying dependency on solution variable. 

Many Thanks 
Pushkar

**Comment by lululxvi** [comment]
See FAQ "Q: Solve inverse problems with unknown parameters/fields in the PDEs or initial/boundary conditions." and "Q: A standard network doesn’t work for me, and I want to implement a network with a special structure/property."

**Reply by ppandit95** [reply]
Thanks Dr Lu @lululxvi for your reply, I just recently discovered the FAQ section which super helpful and also thankful to you for DeepXDE as it provides a great opportunity to solve inverse problems.
